### PR TITLE
Fix: Retrieval of hardware and version from AirOS 8.x

### PR DESCRIPTION
**Pull request by corny**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`

I don't have old AirOS devices to test against.

### CODE REVIEW

**Comment by laf** [comment]
How sure are you that the OID you've removed isn't used in older / other firmware versions?

It's probably worth converting this to multi_gets and querying both OIDs.

**Comment by corny** [comment]
As I wrote I don't have any possibility to test it against old devices. Thanks for mentioning `multi_gets`. I will adjust my code to query both OIDs.

**Comment by laf** [comment]
Give us a shout if you need help.

`includes/polling/os/tpdin.inc.php` has an example of use.

**Comment by corny** [comment]
Thanks a lot!

**Comment by corny** [comment]
```php
snmp_get_multi_oid($device,"dot11manufacturerProductName.5 dot11manufacturerProductName.10", '-OQUs', 'IEEE802dot11-MIB');
```
always returns an empty result. Now I query product name and version with `10` and then with `5`.

File: includes/polling/os/airos.inc.php [hunk 1]
```diff
@@ -1,8 +1,16 @@
 <?php
 
-$hardware = 'Ubiquiti '.trim(snmp_get($device, 'dot11manufacturerProductName.5', '-Ovq', 'IEEE802dot11-MIB'));
+foreach (array(10,5) as $i) {
+    $result = snmp_get_multi_oid($device, "dot11manufacturerProductName.$i dot11manufacturerProductVersion.$i", '-OQUs', 'IEEE802dot11-MIB');
 
-$version         = trim(snmp_get($device, 'dot11manufacturerProductVersion.5', '-Ovq', 'IEEE802dot11-MIB'));
-list(, $version) = preg_split('/\.v/', $version);
+    if (is_array($result)) {
```
Comment (laf) [inline includes/polling/os/airos.inc.php]: I updated this to is_array(), please check this still works.

**Comment by corny** [comment]
`$result` is always an array and it is never empty. Did I ever say I dislike PHP?
I used `print_r()` to inspect the value of `result`.

Invalid result:
```
Array
(
    [] => 
)
```
Valid result:
```
Array
(
    [dot11manufacturerProductName.10] => NanoBeam 5AC 19
    [dot11manufacturerProductVersion.10] => XC.qca955x.v8.1.4.34481.170608.1728
)
```
That means `$result` has to contain exactly two elements.

**Comment by laf** [comment]
Brill, thanks for the PR and sorry I messed it up a little :)

**Comment by corny** [comment]
My first attempt didn't work either. That's just PHP ^^

**Pull request merged**